src/enc/brotli_bit_stream.rs

```diff
@@ -846,21 +846,15 @@ fn BrotliStoreHuffmanTreeOfHuffmanTreeToBitMask(
         }
     }
     BrotliWriteBits(2, skip_some, storage_ix, storage);
-    {
-        let mut i: u64;
-        i = skip_some;
-        while i < codes_to_store {
-            {
-                let l: usize = code_length_bitdepth[(kStorageOrder[i as usize] as usize)] as usize;
-                BrotliWriteBits(
-                    kHuffmanBitLengthHuffmanCodeBitLengths[l],
-                    kHuffmanBitLengthHuffmanCodeSymbols[l] as u64,
-                    storage_ix,
-                    storage,
-                );
-            }
-            i = i.wrapping_add(1);
-        }
+
+    for i in skip_some..codes_to_store {
```

Comment:
What if skip_some > codes_to_store...that would panic, right?
please fix or revert